tests/test_airpyllution.py

```diff
@@ -6,7 +6,7 @@
 from airpyllution.utils import *
 
 
-def mocked_requests_get_pollution_history(*args, **kwargs):
+def mocked_requests_get_pollution(*args, **kwargs):
     class MockResponse:
         def __init__(self, json_data, status_code):
             self.json_data = json_data
```

Comment:
Would you be able to delete the print statement on line 17 - that's my bad, shouldn't have left it in there

Reply:
Done 😊